flight/Modules/Actuator/actuator.c

```diff
@@ -338,7 +338,7 @@ static void actuator_task(void* parameters)
 float process_mixer(const int index, const float curve1, const float curve2,
 		ActuatorDesiredData *desired)
 {
-	int8_t *vector = get_mixer_vec(index);
+	typeof(&(*mixerSettings.Mixer1Vector)) vector = get_mixer_vec(index);
```

Comment:
Actually, @mlyle, I'm not quite sure what you're suggesting. The functions can't return ```typeof(mixerSettings.Mixer1Vector)``` because it's an array type. So are you suggesting that only at the point of L341 that be switched to ```typeof (mixerSettings.Mixer1Vector) (*vector)  = get_mixer_vec(index);```? I don't think this works, and the compiler seems to agree.